test/box/fix-memory-corruption.test.lua

```diff
@@ -0,0 +1,23 @@
+env = require('test_run')
+fiber = require('fiber')
+net_box = require('net.box')
+
+test_run = env.new()
+
+test_run:cmd('create server test_server with script="box/fix-memory-corruption-server.lua"')
+test_run:cmd('create server test_client with script="box/fix-memory-corruption-client.lua"')
+test_run:cmd('start server test_server')
+count = 0
+test_run:cmd("setopt delimiter ';'")
+while test_run:grep_log('default', 'Segmentation fault') == nil and count < 20 do
+    os.execute("./fix-memory-corruption-client.lua&")
+    count = count + 1
+    fiber.sleep(0.1)
```

Comment:
Why do you need this sleep?

Reply:
We are just waiting here to make sure
that none of the clients crashed with segmentation fault

I write this comment it test